authors/apps/highlights/response_messages.py

```diff
@@ -0,0 +1,20 @@
+HL_VALIDATION_MSGS = {
+    "END_LESS_THAN_START": ("The start index is greater than the end index."
+                            " Cannot highlight with invalid indexes."),
+    "OUT_OF_RANGE": "Highlight is out of the article body length",
+    "HIGHLIGHT_UPDATED": "Highlight has been updated.",
+    "HIGHLIGHT_ADDED": "Highlight has been added.",
+    "HIGHLIGHT_REMOVED": "The highlight has been removed."
+}
+
+HIGHLIGHT_MSGS = {
+    "HIGHLIGHT_UPDATED": "Highlight has been updated.",
+    "ARTICLE_NOT_FOUND": ("Could not highlight because ",
+                          "the article could not be found."),
+    "HIGHLIGHT_REMOVED": "Highlight removed.",
```

Comment:
"HIGHLIGHT_UPDATED",  "HIGHLIGHT_ADDED" & "HIGHLIGHT_REMOVED" have been repeated in `HL_VALIDATION_MSGS`

Reply:
Removed duplicates from HL_VALIDATION_MSGS.